go/vt/vttablet/sandboxconn/sandboxconn.go

```diff
@@ -400,7 +400,7 @@ func (sbc *SandboxConn) BeginStreamExecute(ctx context.Context, target *querypb.
 	if err != nil {
 		return 0, nil, err
 	}
-	err = sbc.StreamExecute(ctx, target, sql, bindVariables, transactionID, options, callback)
+	err = sbc.StreamExecute(ctx, target, sql, bindVariables, transactionID, 0, options, callback)
```

Comment:
don't we wan to pass on the reserved id?

Reply:
nice catch, BeginStreamExecute needs reserved connection handling.